src/main/java/org/dependencytrack/notification/publisher/JiraPublisher.java

```diff
@@ -0,0 +1,179 @@
+                    context.put("subjectJson", NotificationUtil.toJson(subject));
+                } else if (notification.getSubject() instanceof AnalysisDecisionChange) {
+                    final AnalysisDecisionChange subject = (AnalysisDecisionChange) notification.getSubject();
+                    context.put("subject", subject);
+                    context.put("subjectJson", NotificationUtil.toJson(subject));
+                } else if (notification.getSubject() instanceof BomConsumedOrProcessed) {
+                    final BomConsumedOrProcessed subject = (BomConsumedOrProcessed) notification.getSubject();
+                    context.put("subject", subject);
+                    context.put("subjectJson", NotificationUtil.toJson(subject));
+                }
+            }
+            // Jira addition:
+            context.put("jiraProjectKey", jiraProjectKey);
+
+            try (Writer writer = new StringWriter()) {
+                template.evaluate(writer, context);
+                return writer.toString();
+            } catch (IOException e) {
+                Logger.getLogger(this.getClass()).error("An error was encountered evaluating template", e);
+                return null;
+            }
+        }
+    }
+
+    @Override
+    public void publish(final String publisherName, final PebbleTemplate template, final Notification notification, final JsonObject config) {
+        final Logger logger = Logger.getLogger(this.getClass());
+        logger.info("Preparing to publish JIRA notification");
+        if (config == null) {
+            logger.warn("No configuration found. Skipping notification.");
+            return;
+        }
+        
+        try (QueryManager qm = new QueryManager()) {
+            jiraUsername = qm.getConfigProperty(JIRA_USERNAME.getGroupName(), JIRA_USERNAME.getPropertyName()).getPropertyValue();
+            final String password = qm.getConfigProperty(JIRA_PASSWORD.getGroupName(), JIRA_PASSWORD.getPropertyName()).getPropertyValue();
+            jiraPassword = (password != null) ? DataEncryption.decryptAsString(password) : null;
+            final String configDestination = config.getString("destination");
+            URL url;
+            String destination;
+            url = new URL(configDestination);
+            destination = url.getProtocol()+"://"+url.getAuthority()+"/rest/api/2/issue";
+            final String[] pathElements = url.getPath().split("/");
+            if(pathElements.length == 0) {
+                logger.error("No Jira Project given");
+                return;
+            }
+            jiraProjectKey = pathElements[pathElements.length-1];
+            final String content = prepareTemplate(notification, template);
+            if (destination == null || content == null) {
+                logger.warn("A destination or template was not found. Skipping notification");
+                return;
+            }
+            logger.info("Template: "+content);
```

Comment:
Please remove or change log level to `debug`.

Reply:
removed